CHANGELOG.md

```diff
@@ -21,6 +23,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  - Built Brokerpaks now have a name of `{name}-{version}.brokerpak` as defined by the manifest rather than the name of the parent directory.
  - Services inside Brokerpaks now have a file name that includes their CLI friendly name to help differentiate them.
  - The `pak build` command now includes progress logs.
+ - Instance label prefixes are now `cf-` rather than `pcf-`.
```

Comment:
Nice catch, It might be worth doing a quick check for any other unnecessary references to pcf, if you haven't already.

Reply:
That's a good idea. I see a few more mostly in the default auto-generated names of services. I'll add a ticket to replace these.